<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gradeWindow</class>
 <widget class="QWidget" name="gradeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="mainImage">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="typeGradeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Тип оцінки&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateTakeLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Дата виставлення&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="gradeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Оцінка&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="subjectLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Предмет&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="whoTakeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Отримувач&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Зберегти</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeGradeComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Оберіть тип оцінки</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Семестрова</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Тематична</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Модульна</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Семінар</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Поточна</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Екзамен</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Іспит</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Річна</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="gradeSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>135</width>
     <height>21</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Скасувати</string>
   </property>
  </widget>
  <widget class="QComboBox" name="subjectComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Оберіть предмет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>rew</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>wrewr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>wer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>wer</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="whoTakeComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Оберіть отримувача</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>rew</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>wrewr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>wer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>wer</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="takeDateEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>135</width>
     <height>21</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>9</month>
     <day>14</day>
    </datetime>
   </property>
   <property name="minimumDate">
    <date>
     <year>2000</year>
     <month>9</month>
     <day>14</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="okLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>125</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>%status_label%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="groupComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Оберіть групу отримувача</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>rew</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>wrewr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>wer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>wer</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="subjectLabel_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Група отримувача&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>saveButton</zorder>
  <zorder>mainImage</zorder>
  <zorder>typeGradeLabel</zorder>
  <zorder>dateTakeLabel</zorder>
  <zorder>gradeLabel</zorder>
  <zorder>subjectLabel</zorder>
  <zorder>whoTakeLabel</zorder>
  <zorder>typeGradeComboBox</zorder>
  <zorder>gradeSpinBox</zorder>
  <zorder>cancelButton</zorder>
  <zorder>subjectComboBox</zorder>
  <zorder>whoTakeComboBox</zorder>
  <zorder>takeDateEdit</zorder>
  <zorder>okLabel</zorder>
  <zorder>groupComboBox</zorder>
  <zorder>subjectLabel_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
